lib/src/components/NxFieldset/NxFieldset.tsx

```diff
@@ -19,8 +19,8 @@ const NxFieldset = forwardRef<HTMLFieldSetElement, Props>(
         <fieldset className={classNames} ref={ref} { ...attrs }>
           <legend className={legendClassnames}>
             <span className="nx-legend__text">{label}</span>
-            { sublabel && <span className="nx-sub-label">{sublabel}</span> }
           </legend>
+          { sublabel && <span className="nx-sub-label">{sublabel}</span> }
```

Comment:
You changed the one in the form layout example to a `<div>`, but this is still a `<span>`

Reply:
> You changed the one in the form layout example to a `<div>`, but this is still a `<span>`

Drat I missed that: https://github.com/sonatype/sonatype-react-shared-components/pull/266/commits/243d13fe6b6d4c3040f01092a2658888ae84204a